packages/material-ui/src/MenuItem/MenuItem.js

```diff
@@ -176,6 +232,7 @@ const MenuItem = React.forwardRef(function MenuItem(inProps, ref) {
   };
 
   const classes = useUtilityClasses(props);
+  const hookClasses = useStyles({ theme });
```

Comment:
doesnt `makeStyles` have access to the theme already?

Reply:
You would think. I actually ran into a super weird issue related to makeStyles not calling its function argument with the theme from the ThemeProvider and the test suite failed to even run citing this issue. I was already grabbing the theme with `useTheme` for a conditional style application (`theme.direction` for rtl support) and the only way I could get the thing happy was to do this instead of grabbing `theme` from the first function arg, like ususal. /shrug

Reply:
It seems like the bigger issue here is that we're using makeStyles at all. I'd like to completely do away with that and use the new emotion way of doing things, but I couldn't get it to work well after a fair bit of trying and I figured I'd just go with what works for now and we could start getting feedback on the main MR once we're in a happy place.

Reply:
ah ok, ya I can agree with both points but I figured I'd ask!